setup.py

```diff
@@ -29,6 +36,8 @@
         'qualtricssurvey': 'qualtricssurvey',
     },
     package_data={
+        '': [
+        ],
```

Comment:
why this empty item?

Reply:
This could probably go away.
We had previously used the empty stanza for the `package.json`;
this may just be me cargo-culting myself...